packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -88,7 +88,7 @@ import VaTimePicker from '../va-time-picker/VaTimePicker.vue'
 import VaInput from '../va-input/VaInput.vue'
 import VaIcon from '../va-icon/VaIcon.vue'
 import VaDropdown from '../va-dropdown/VaDropdown.vue'
-import VaDropdownContent from '../va-dropdown/VaDropdownContent/VaDropdownContent.vue'
+import VaDropdownContent from '../va-dropdown/'
```

Comment:
is `VaDropdownContent` default import?
redundant `/`

Reply:
Nice catch! Didn't notice that.